Update navigation for Insights (#3960)
theforeman/foreman-documentation: guides/common/modules/ref_access-to-information-from-insights-in-project.adoc

You can access the additional information available for hosts from Red{nbsp}Hat Insights in the following places in the {ProjectWebUI}:

> * Navigate to *Insights* > *Recommendations* where the vertical ellipsis next to the *Remediate* button provides a *View in Red{nbsp}Hat Insights* link to the general recommendations page.

Reviewer: FYI. RHSSG says not to use 'vertical ellipsis'. I understand this is outside the scope of this PR.

> * Navigate to *Insights* > *Recommendations* where the vertical ellipsis next to the *Remediate* button provides a *View in Red{nbsp}Hat Insights* link to the general recommendations page.

Reviewer: ```suggestion
* Navigate to *Insights* > *Recommendations*, click the vertical ellipsis next to the *Remediate* button, and select *View in Red{nbsp}Hat Insights*.
```
Not sure whether this will help. :-(
